### ISSUE TITLE
[Feature Request] Option to show shares on main page

### ISSUE THREAD

**Issue by dakiba** [sissbruecker/linkding#378]
It would be nice if we could see the shares on the main page, or at least have the search and tags work with the shares as well. My girlfriend and I want to use Linkding for our links, but we only see the option of using one account for our purpose. Ultimately, we don't want to share all of the links...

**Comment by sissbruecker** [comment]
Yeah, I left that out in the initial implementation to keep it simple. I also remember doing lots of thinking around possible conflicts, but can't remember what these were. One thing that comes to mind is that it can be annoying when someone shares all sorts of bookmarks that now start cluttering up your private bookmark list. Maybe there should be a filter for shared bookmarks. Apart from that the performance for the query that loads shared bookmarks is not great, and could use some tuning.

**Comment by silbaer** [comment]
I am new to linkding and the first I missed when I start playing with shared bookmarks, is the integration in the main page. 
Cluttering the private bookmarks is a point, but an the other hand I want to simple search **all** bookmarks in **one** search. 
At the moment, I have to go to my bookmarks, enter the search, scroll through the result and than I have to switch to shared bookmarks (with *two* clicks), and do the same **again**.

I wish there is a way to toggle between "my bookmarks" and "all bookmarks" (displaying my **and** shared bookmarks).

**Comment by Neurrone** [comment]
It took me an embarrassingly long time to figure out how to even view shared bookmarks. Would it be possible to have a link to the `/bookmarks/shared` page on the main bookmarks list so that it is discoverable? The only way I found it was going into the settings of the user, which has a link to the shared bookmarks page there.

**Comment by Neurrone** [comment]
Ok, I just realized that the global settings has a toggle to control whether the login form or the list of publicly shared bookmarks is shown.
